clients/commercetools/index.js

```diff
@@ -0,0 +1,26 @@
+require('isomorphic-fetch');
+import { createClient } from '@commercetools/sdk-client';
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http';
+import { createAuthMiddlewareForClientCredentialsFlow } from '@commercetools/sdk-middleware-auth';
+
+const client = () =>
+  createClient({
+    middlewares: [
+      createAuthMiddlewareForClientCredentialsFlow({
+        host:
+          process.env.COMMERCETOOLS_AUTH_HOST ||
+          'https://auth.commercetools.co',
```

Comment:
I would extract this kind of things to a constants file. We are using it now in two places 👍